core/src/i18n/en.yaml

```
@@ -232,16 +231,20 @@ clusters:
     title-current-cluster: Cluster Overview
     version: Version
   storage:
-    changed-message: The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to {{targetStorage}}.
+    changed-message: >-
+      The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to
```

```suggestion
      The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to {{targetStorage}}.
```

Why split that sentence to two lines? Not good for translators.